code/Makefile

```diff
@@ -179,9 +178,9 @@ BATCHTERM=dumb
 #--- Actual targets ---#
 ########################
 
-# Default `make` will just run examples and test them against stable.
-# See `test` target for more details.
-all: test ##@Examples Run examples and test against stable.
+# Default `make` will just run examples and test them against stable. See `test`
+# target for more details.
+all: test test_website ##@Examples Run examples and test against stable.
```

Comment:
`test_website` is a good target to add here because it makes `make all` and `make pr_ready` test the website against the stabilized website artifacts as well.